seeds/data/profiles.json

```diff
@@ -506,16 +506,11 @@
   {
     "permissions": [ { "establishmentId": 8201 } ],
     "title": "Prof",
-    "firstName": "Mano",
-    "lastName": "Yanele",
+    "firstName": "Hasnø",
+    "lastName": "Pil",
     "dob": "1950-03-22",
-    "email": "myaneleo@bizjournals.com",
-    "telephone": 4996576030,
-    "pil": {
-        "issueDate": "2014-09-26",
-        "licenceNumber": "CF-498239",
-        "conditions": "In congue. Etiam justo. Etiam pretium iaculis justo.\n\nIn hac habitasse platea dictumst. Etiam faucibus cursus urna. Ut tellus."
-    }
+    "email": "hasnø.pil@croydon.ac.uk",
```

Comment:
Can I request that all emails that we use for test accounts that actually do things are `*@example.com` (or guerillamail, or domains we own).

`croydon.ac.uk` is an actual domain for an actual organisation, and at some point we will enable notifications, an it's be good not to spam the crap out of them.

Reply:
Ah, assumed it was a made up one (Uni of Croydon) cause it's used throughout that file.

Reply:
I suspect someone (let's call them "me") assumed the same at some point in the past. Turns out _that person_ was wrong.